docs/governance/community-guidelines/numbering-format.md

```diff
@@ -0,0 +1,13 @@
+# Numbering Format
+
+## Taxonomy
+
+This is the proposed numbering format used in the taxonomy service features.
+
+`CCC.TX.CAT.XX.YYY`
+
+- **CCC** - Common Coloud Control. Fixed
+- **TX** - Taxonomy. Fixed
+- **CAT** - Refer to Category. Valid values NW (Network Services), COMP (Computing Services), DB (Database Services), STR (Storage Services)
```

Comment:
```suggestion
- **CAT** - Refer to Category. Valid values: NW (Network Services), COMP (Computing Services), DB (Database Services), STR (Storage Services)
```

Is this a comprehensive list?

Reply:
No. These are the only categories we have started work on at the moment